README.md

```diff
@@ -1,182 +1,163 @@
 # JavaScript Data Types
-
-In this lesson, we'll cover some of the most commonly used data types in JavaScript.
+In this lesson, we'll cover all of the basic data types in JavaScript.
 
 ## Objectives
-1. Define "data type"
-2. Explain JavaScript's data types
-3. Use the `typeof` operator
+1. Define 'data type'.
+2. Introduce the `typeof` operator.
```

Comment:
Somewhat nit=picky but I really like using the Students Will Be Able To... format of Objectives. So everything in this list should be able to be prefixed with "After this lesson, students will be able to....". I find it grounds the objectives a bit better.

Instead of "Introduce the `typeof`operator" I would say: "Determine the type of a javascript variable".